### PR TITLE
adding starting django code

### PR DESCRIPTION
**Pull request by jkim10**

### CODE REVIEW

**Comment by jkim10** [comment]
Need this to start automating

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,6 +1,9 @@
 {
     "python.linting.pylintEnabled": false,
     "python.linting.flake8Enabled": true,
+    "python.linting.flake8Args": [
+        "--max-line-length=100",
```
Comment (ceguiluzrosas) [inline .vscode/settings.json:5]: Had to update the line length because some lines were off by a few characters.

**Comment by ceguiluzrosas** [comment]
Followed this quick-start guide: https://www.django-rest-framework.org/tutorial/quickstart/

**Review by ceguiluzrosas — APPROVED** [review]
LGTM 👍🏽

**Pull request merged**